### PR TITLE
experience plugin: avoid deprecated code and friends

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/plugin/encounter/EncounterPlugin.java [hunk 1]
```diff
@@ -38,15 +38,6 @@
...
 
-import gmgen.GMGenSystem;
-import gmgen.GMGenSystemView;
-import gmgen.io.ReadXML;
-import gmgen.io.VectorTable;
-import gmgen.plugin.InitHolderList;
-import gmgen.plugin.PcgCombatant;
-import gmgen.plugin.dice.Dice;
-import gmgen.pluginmgr.messages.AddMenuItemToGMGenToolsMenuMessage;
-import gmgen.pluginmgr.messages.RequestAddTabToGMGenMessage;
```
Comment (thpr) [inline code/src/java/plugin/encounter/EncounterPlugin.java]: This is an imports ordering problem.  gmgen is part of pcgen and should be in the pcgen block.

File: code/src/java/plugin/experience/ExperienceAdjusterModel.java [hunk 2]
```diff
@@ -19,16 +19,18 @@
 
 import java.io.File;
 import java.util.Observable;
+import java.util.stream.IntStream;
+
+import pcgen.core.SettingsHandler;
+import pcgen.util.Logging;
+import plugin.experience.gui.PreferencesExperiencePanel;
```
Comment (thpr) [inline code/src/java/plugin/experience/ExperienceAdjusterModel.java]: Same as above

File: code/src/java/plugin/experience/ExperienceAdjusterPlugin.java [hunk 3]
```diff
@@ -31,15 +32,6 @@
...
 
-import gmgen.GMGenSystem;
-import gmgen.GMGenSystemView;
-import gmgen.plugin.Combatant;
-import gmgen.plugin.InitHolderList;
-import gmgen.pluginmgr.messages.AddMenuItemToGMGenToolsMenuMessage;
-import gmgen.pluginmgr.messages.CombatHasBeenInitiatedMessage;
-import gmgen.pluginmgr.messages.FileMenuSaveMessage;
-import gmgen.pluginmgr.messages.RequestAddPreferencesPanelMessage;
-import gmgen.pluginmgr.messages.RequestAddTabToGMGenMessage;
```
Comment (thpr) [inline code/src/java/plugin/experience/ExperienceAdjusterPlugin.java]: Same as above

File: code/src/java/plugin/experience/gui/ExperienceAdjusterView.java [hunk 4]
```diff
@@ -497,12 +497,12 @@ public void valueChanged(ListSelectionEvent e)
 		add(jPanel6);
 	}
 
-	private double calculateRealValue(int i)
+	private static double calculateRealValue(int i)
```
Comment (thpr) [inline code/src/java/plugin/experience/gui/ExperienceAdjusterView.java]: Either this method is unused and should be deleted, or you are now violating the access rules for static methods by accessing them from an instance.

**Comment by grimreaper** [comment]
My settings file had `gmgem` instead of `gmgen. Fixing. thanks.

File: code/src/java/plugin/experience/gui/ExperienceAdjusterView.java (hunk 4 above)
Comment (grimreaper) [reply code/src/java/plugin/experience/gui/ExperienceAdjusterView.java]: It was being accessed without an explicit name, so it was allowed. Added `ExperienceAdjusterView.` as a prefix.

**Pull request merged**